src/Pact/Gas/Table.hs

```diff
@@ -229,36 +229,37 @@ tableGasModel :: GasCostConfig -> GasModel
 tableGasModel gasConfig =
```

Comment:
Should we take one of the gas table entries and change its units from `Gas` to `MilliGas`, in order to exercise the rest of this PR and make sure that the computed gas is the same?

In other words, is the PR exercised enough if we keep all of the old gas table entries specified in coarse `Gas`, and then convert from millis back to course gas at the end?

In yet other words, with the current gas table, do we expect all our gas calculations to be some multiple of 1000?

Reply:
> In other words, is the PR exercised enough if we keep all of the old gas table entries specified in coarse Gas, and then convert from millis back to course gas at the end?

In the current gas table we expect this. I actually left this as is to ensure migration is smoother and I wouldn't get it wrong. It's less code to insert a conversion.

We do not expect _all calcs_ to be multiples of 1000 gas, but we do expect all _old gas calculations_ to be such. If you note the return type of `runGasModel` though, it's in `MilliGas`